### ISSUE TITLE
Warning about peer dep eslint with io.js

### ISSUE THREAD

**Issue by albertosantini** [Gillespie59/eslint-plugin-angular#84]
Just a note about peer dep eslint with io.js (1.6.3).

```
λ npm up
npm WARN peerDependencies The peer dependency eslint@>=0.8.0 included from eslint-plugin-angular will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
eslint-plugin-angular@0.0.7 node_modules\eslint-plugin-angular
```

**Comment by Gillespie59** [comment]
Thanks Alberto.

**Comment by Gillespie59** [comment]
Hi, 
I think this is not an issue. This is just a warning for the upcoming nom@3. We do not have to do push any changes on our plugin. 

In the next version of npm, the peerDependency won't be installed automatically like with npm@2. THe end-user will have to install it explicitly in his project. 

This warning is just a message to prepare everybody ot this big change. 

Manu

**Comment by Gillespie59** [comment]
We can maybe add a simple comment in our README.md file

**Comment by albertosantini** [comment]
Agreed.
